Dockerfile

```diff
@@ -4,4 +4,12 @@
 # https://github.com/QGreenland-Net/ogdc-helm/issues/31
 FROM ghcr.io/osgeo/gdal:ubuntu-small-latest
 # We use and wget to fetch data from remote sources.
-RUN apt update && apt install -y wget rsync
+RUN apt update && apt install -y wget rsync pip python3-venv
+
+WORKDIR /ogdc_runner/
+COPY . /ogdc_runner
+RUN python -m venv .venv && . ./.venv/bin/activate && pip install --editable .
+
+EXPOSE 8000
+
+CMD . ./.venv/bin/activate && fastapi dev --port 8000 --host 0.0.0.0 src/ogdc_runner/service.py
```

Comment:
I'll update the default CMD here in a future PR (probably addressing https://github.com/QGreenland-Net/ogdc-runner/issues/108)